packages/gestalt/src/ListText.js

```diff
@@ -0,0 +1,36 @@
+// @flow strict
+import { Children, type Element, type Node } from 'react';
+import Text from './Text.js';
+import styles from './List.css';
+import { useColorScheme } from './contexts/ColorSchemeProvider.js';
+
+type Props = {|
+  text: string | Element<typeof Text>,
+|};
+
+export default function ListText({ text }: Props): Node {
+  if (!text) {
```

Comment:
I added this here because when label is not passed, the error thrown isn't great. In a world with Flow, that's a no brainer. But in our code examples, I had issues realizing I was missing labels. Added this as a safety guard.

Reply:
Makes sense. Consider leaving that context as a comment in the code here